docs/framework/guides/tutorials/implementing-an-inline-widget.md

```diff
@@ -734,7 +734,7 @@ ClassicEditor
 			types: [ 'date', 'color', 'first name', 'surname' ]                             // ADDED
 		}
 	} )
-	// ...
```

Comment:
We should keep those ellipses below the comment, as shown here: https://github.com/cksource/ckeditor5-internal/issues/2307.